packages/inventory-general-info/src/CollectionCard/CollectionCard.js

```diff
@@ -1,16 +1,36 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
+
+import { Tooltip } from '@patternfly/react-core';
+
 import LoadingCard from '../LoadingCard';
 import { collectionInformationSelector } from '../selectors';
 import DateFormat from '@redhat-cloud-services/frontend-components/DateFormat';
 
+const VersionTooltip = ({ egg, client }) => (
+    <Tooltip
+        content={
+            <React.Fragment>
+                <p>RPM version: { egg || 'Not available' }</p>
+                <p>Dynamic update version: { client || 'Not available' }</p>
```

Comment:
This should be flipped, the egg is `Dynamic` and `client` is RPM version.
```suggestion
                <p>RPM version: { client || 'Not available' }</p>
                <p>Dynamic update version: { egg || 'Not available' }</p>
```